src/targets/github.ts

```diff
@@ -122,13 +127,23 @@ export class GitHubTarget extends BaseTarget {
       };
     }
 
+    const { data: latestRelease } = await this.github.repos.getLatestRelease({
```

Comment:
this crashes when there are no releases in a repository -- can you fix this regression?

Reply:
I will look into it!